modules/core/src/test/java/org/locationtech/jts/geom/CoordinateSequencesTest.java

```diff
@@ -67,4 +125,259 @@ private static CoordinateSequence createTestSequence(CoordinateSequenceFactory c
     return cs;
   }
 
+  private static CoordinateSequence createRandomTestSequence(CoordinateSequenceFactory csFactory, int size, int dim,
```

Comment:
It's maybe a style thing, but I prefer to not use randomly generated data in unit tests. It makes it hard to reproduce behaviour.  (Its fine for performance testing, though - but JTS has a different set of packages for this).

Reply:
FYI https://labs.carrotsearch.com/randomizedtesting.html is some cool test tooling to help with randomizations -- most importantly to make a randomized test _easily repeatable_.  Lucene/Solr uses this extensively and I've become a big proponent of the methodology.

If randomized tests aren't acceptable in JTS, I suggest @FObermaier merely tweak this test he already invested time in to simply use a constant seed -- `new Random(0)`.